webapp/src/main/java/com/box/l10n/mojito/rest/scheduledjob/ScheduledJobWS.java

```diff
@@ -35,18 +39,159 @@ public class ScheduledJobWS {
   static Logger logger = LoggerFactory.getLogger(ScheduledJobWS.class);
   private final ScheduledJobRepository scheduledJobRepository;
   private final ScheduledJobManager scheduledJobManager;
+  private final ScheduledJobStatusRepository scheduledJobStatusRepository;
+  private final ScheduledJobTypeRepository scheduledJobTypeRepository;
 
   @Autowired
   public ScheduledJobWS(
-      ScheduledJobRepository scheduledJobRepository, ScheduledJobManager scheduledJobManager) {
+      ScheduledJobRepository scheduledJobRepository,
+      ScheduledJobManager scheduledJobManager,
+      ScheduledJobStatusRepository scheduledJobStatusRepository,
+      ScheduledJobTypeRepository scheduledJobTypeRepository) {
     this.scheduledJobRepository = scheduledJobRepository;
     this.scheduledJobManager = scheduledJobManager;
+    this.scheduledJobStatusRepository = scheduledJobStatusRepository;
+    this.scheduledJobTypeRepository = scheduledJobTypeRepository;
   }
 
   private final ResponseEntity<ScheduledJobResponse> notFoundResponse =
       createResponse(
           HttpStatus.NOT_FOUND, ScheduledJobResponse.Status.FAILURE, "Job doesn't exist", null);
 
+  @RequestMapping(method = RequestMethod.POST, value = "/api/jobs/create")
```

Comment:
```suggestion
  @RequestMapping(method = RequestMethod.POST, value = "/api/jobs")
```
We should keep the endpoint name as `/api/jobs`, POST will indicate that a new resource is being created, will share some information on this offline